src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/FeatureFlag/index.html.twig

```diff
@@ -0,0 +1,73 @@
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain 'Admin.Advparameters.Feature' %}
+
+{% form_theme featureFlagsForm 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit.html.twig' %}
+
+{% block content %}
+  <div class="row justify-content-center">
+    {{ form_start(featureFlagsForm, {attr : {class: 'form col', id:'feature-flag-form'}, action: path('admin_feature_flags_index') }) }}
+    {% block feature_flag_form %}
+      <div class="card" id="configuration_fieldset_general">
+        <h3 class="card-header">
+          <i class="material-icons">settings</i> {{ 'Experimental features'|trans }}
+        </h3>
+
+        <div class="card-block row">
+          <div class="card-text">
+            <div class="alert medium-alert alert-warning" role="alert">
+              {{ 'Testing a feature before its official release can be exciting. However, you must be aware of the potential risks of such experiments:'|trans({}, 'Admin.Advparameters.Notification') }}
+              <ul>
+                <li>{{ 'Experimental features are still under development. Enabling them could therefore have unintended consequences and cause data loss.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+                <li>{{ 'In any case, you should never experiment in production.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+              </ul>
+            </div>
+            {{ form_widget(featureFlagsForm) }}
+          </div>
+        </div>
+        <div class="card-footer">
+          <div class="d-flex justify-content-end">
+            <button type="submit" id="submit-btn-feature-flag" class="btn btn-primary"
+                    data-modal-title="{{ 'Are you sure you want to enable this experimental feature?'|trans({}, 'Admin.Advparameters.Notification') }}"
+                    data-modal-message="{{ 'You are about to enable a feature that is not stable yet. This should only be done in a test environment or in full knowledge of the potential risks.'|trans({}, 'Admin.Advparameters.Notification') }}"
+                    data-modal-apply="{{ 'Enable'|trans({}, 'Admin.Actions') }}"
+                    data-modal-cancel="{{ 'Cancel'|trans({}, 'Admin.Actions') }}"
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/abed71ba1f7efc15ee413a685ce184c2c52e6b90